<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1562</width>
    <height>1068</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_rec_rar">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>130</y>
      <width>221</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>right acutal rpm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_lar">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>130</y>
      <width>211</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>left actual rpm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_rec">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>281</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>right encoder count </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_volt">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>volt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_ampere">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>129</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>apmere</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_estop">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>240</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>estop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_bump">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>129</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>bump</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_cliff">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>129</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>cliff</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_ir">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>350</y>
      <width>61</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>ir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_rdr">
    <property name="geometry">
     <rect>
      <x>1054</x>
      <y>700</y>
      <width>231</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>right desire rpm </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_chk">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>458</y>
      <width>129</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>checksum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_receive_data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>291</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Receive Data ==&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_yaw_vel">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>456</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>yaw velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_yaw_heading">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>456</y>
      <width>191</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>yaw heading</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_lec">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>129</y>
      <width>281</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>left encoder count </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio1">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>241</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>led1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio2">
    <property name="geometry">
     <rect>
      <x>758</x>
      <y>242</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>led2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio3">
    <property name="geometry">
     <rect>
      <x>635</x>
      <y>242</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>led3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>242</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>led4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>242</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>gpio5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>242</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>gpio6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>242</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>gpio7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>242</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>gpio8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_transimt_data">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>630</y>
      <width>331</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;== Transmit Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_transimt_hz_value">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>630</y>
      <width>71</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_ldr">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>702</y>
      <width>231</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>left desire rpm </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_led1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>809</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>led1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_led6">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>810</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>gpio6</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_led8">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>810</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>gpio8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_estop">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>808</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>estop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_led4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>810</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>led4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_led7">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>810</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>gpio7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_led2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>810</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>led2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_led3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>810</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>led3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_led5">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>810</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>gpio5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_chk">
    <property name="geometry">
     <rect>
      <x>1395</x>
      <y>811</y>
      <width>129</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>checksum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_transimt_hz">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>629</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hz : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_receive_hz">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>67</y>
      <width>61</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hz : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_receive_hz_value">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>68</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_disconnect">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>30</y>
      <width>261</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_rec_value">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>xxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_lec_value">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>169</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>xxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_rar_value">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>170</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>xxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_lar_value">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>170</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>xxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_ampere_value">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>xxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_volt_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>xxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio6_value">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>288</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio1_value">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>287</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio8_value">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>288</y>
      <width>61</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_estop_value">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>286</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio4_value">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>288</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio7_value">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>288</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio2_value">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>288</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio5_value">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>288</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_gpio3_value">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>288</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_cliff_value">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>392</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>xxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_bump_value">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>392</y>
      <width>129</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>xxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_ir_value">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>392</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>xxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_chk_value">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>508</y>
      <width>129</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>xxxxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_yaw_heading_value">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>506</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>xxxxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_rec_yaw_vel_value">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>506</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>xxxxxxxx</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran_chk_value">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>863</y>
      <width>129</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>xxxxxxxx</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_rdr">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>750</y>
      <width>113</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit__transimt_ldr">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>750</y>
      <width>113</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit__transimt_estop">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_gpio1">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_gpio2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_gpio3">
    <property name="geometry">
     <rect>
      <x>673</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_gpio4">
    <property name="geometry">
     <rect>
      <x>783</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_gpio5">
    <property name="geometry">
     <rect>
      <x>911</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_gpio6">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_gpio7">
    <property name="geometry">
     <rect>
      <x>1166</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_transimt_gpio8">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>860</y>
      <width>51</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
